src/V3Delayed.cpp

```diff
@@ -246,8 +262,122 @@ class DelayedVisitor final : public VNVisitor {
 
     // METHODS
 
+    // Return true iff a variable is assigned by both blocking and nonblocking
+    // assignments. Issue BLKANDNBLK error if we can't prove the mixed
+    // assignments are to independent bits and the blocking assignment can be
+    // in combinational logic, which is something we can't safely implement
+    // still.
+    bool checkMixedUsage(const AstVarScope* vscp, bool isPacked) {
```

Comment:
```suggestion
    bool checkMixedUsage(const AstVarScope* vscp, bool isIntegralOrPacked) {
```

As on first reading I wondered why only doing structs.

Reply:
Done, FWIW in my [potentially broken] parlance, 'packed' just means anything that `_[]` bit extract and biwise operator `~`, `&`, `|`, `^` are defined on.